### DISCUSSION TITLE
I want an example to stream recorded audio in ESP-EYE to browser or any of interned media (i.e. cloud)

### DISCUSSION THREAD

**Discussion by misterb0407** [pschatzmann/arduino-audio-tools#902] [Q&A] [answered]
I have a project using ESP-EYE, it has built in microphone already connected by I2S. I think I managed to init the microphone, but the issue is I don't have any mean to test the recorded voice as there is no SD card mounted. I am thinking to stream my recorded audio from the C buffer to any of URL so that  I can hear it back from internet.

Could someone give me if there is example closely resemble what I need here? Thank you!

**Comment by pschatzmann** [comment, marked as answer]
I don't understand why you even ask this question: you can find this information in the existing documentation: https://github.com/pschatzmann/arduino-audio-tools/wiki/Examples

Don't forget the existing issues

**Reply by misterb0407** [reply]
Alright my bad,thanks anyway

**Comment by williamsokol** [comment]
This guy does something similar to what you requested, but it is not for the esp eye and uses external mems microphone instead:
https://www.youtube.com/watch?v=VVGln_-XEgI

**Comment by pschatzmann** [comment]
The examples related to the different communications protocols can be found in [examples-communication](https://github.com/pschatzmann/arduino-audio-tools/tree/main/examples/examples-communication). 
You can e.g. serve the audio via a [http](https://github.com/pschatzmann/arduino-audio-tools/tree/main/examples/examples-communication/http-server).
